extensions/analyticsdx-vscode-templates/src/layout/hovers.ts

```diff
@@ -21,13 +22,6 @@ export class LayoutVariableHoverProvider extends VariableRefHoverProvider {
   }
 
   protected override isSupportedLocation(location: Location) {
-    return (
-      !location.isAtPropertyKey &&
-      location.previousNode?.type === 'string' &&
-      // TODO: make these more specific to the layout type (e.g. only 'center' if SingleColumn)
-      (location.matches(['pages', '*', 'layout', 'center', 'items', '*', 'name']) ||
-        location.matches(['pages', '*', 'layout', 'right', 'items', '*', 'name']) ||
-        location.matches(['pages', '*', 'layout', 'left', 'items', '*', 'name']))
-    );
+    return !location.isAtPropertyKey && location.previousNode?.type === 'string' && matchesLayoutItem(location, 'name');
```

Comment:
These variable name hover's weren't showing on variable items in a GroupBox before, but now they do with the utility method.

Reply:
Ah thanks for fixing and refactoring them. I must've missed these last time :))